pydicer/convert/pet_to_nifti_convert_new.py

```diff
@@ -0,0 +1,409 @@
+import warnings
+from os.path import join
+from datetime import time, datetime
+
+import numpy as np
+from skimage.draw import polygon
+import SimpleITK as sitk
+import pydicom as pdcm
+from pydicom.tag import Tag
+
+
+def convert_dicom_to_nifty(input_filepaths,
+                           output_folder,
+                           modality='CT',
+                           sitk_writer=None,
+                           rtstruct_file=None,
+                           labels_rtstruct=['GTVt'],
+                           patient_weight_from_ct=None,
+                           extension='.nii',

```

Comment:
Never use .nii extension, always use .nii.gz. This adds gzip compression to the files which makes them way smaller!